src/main/pom-parser/pom-parser.js

```diff
@@ -12,6 +12,51 @@ const getCoordinatesFromNode = (node) => {
   };
 };
 
+const getStatusFromRepo = (release) => {
```

Comment:
These elements are actually considered a "RepositoryPolicy" based on the type specified in the [xsd](http://maven.apache.org/xsd/maven-4.0.0.xsd).  

Let's go ahead and update our naming to be consistent with that.

```xml
<xs:element minOccurs="0" name="releases" type="RepositoryPolicy">
  <xs:annotation>
    <xs:documentation source="version">4.0.0+</xs:documentation>
    <xs:documentation source="description">
        How to handle downloading of releases from this repository.
    </xs:documentation>
  </xs:annotation>
</xs:element>
```